### PR TITLE
Remove geometric & other minor reformulations

### PR DESCRIPTION
**Pull request by rueuntal**
I have removed the geometric distribution from `sad-comparisons.py` and `sad-process-db.py`, and done a bit of other minor changes. The scripts have been test-run on Gentry, but not any other data set. I'm submitting the pull request just in case folks want to begin playing with the new ideas for plotting.

### CODE REVIEW

File: sad-process-db.py [hunk 1]
```diff
@@ -14,9 +14,9 @@
 
 # Function to import the AICc results.
 def import_results(datafile):
-    """Imports raw result .csv files in the form: site, S, N, logseries, logseries_untruncated, pln, negbin, geometric, zipf."""
-    raw_results = np.genfromtxt(datafile, dtype = "S15, i8, i8, f8, f8, f8, f8, f8", skip_header = 1, 
-                                names = ['site', 'S', 'N', 'logseries', 'pln', 'negbin', 'geometric','AICc_zipf'], delimiter = ",", missing_values = '', filling_values = '')
+    """Imports raw result .csv files in the form: site, S, N, ogseries_untruncated, pln, negbin, zipf."""
```
Comment (ethanwhite) [inline sad-process-db.py]: Looks like you deleted an extra `l` at the start of what is now `ogseries_untruncated` :smile:

File: sad-process-db.py [hunk 2]
```diff
@@ -24,11 +24,14 @@ def winning_model(data_dir, dataset_name, results):
...
     output_processed = csv.writer(open(data_dir + dataset_name + '_processed_results.csv','wb'))
     # Insert comment line
-    output_processed.writerow(["# 0 = Logseries, 1 = Poisson lognormal, 2 = Negative binomial, 3 = Geometric series, 4 = Zipf distribution"])
+    output_processed.writerow(["# 0 = Logseries, 1 = Poisson lognormal, 2 = Negative binomial, 3 = Zipf distribution"])
     
     # Insert header
     output_processed.writerow(['dataset', 'site', 'S', 'N', "model_code", "model_name", "AICc_weight"])
    
+    # Create dictionary of models and their corresponding codes
+    models = {0: 'Logseries', 1:'Poisson lognormal', 2:'Negative binomial', 3:'Negative binomial'}
```
Comment (ethanwhite) [inline sad-process-db.py]: I think it should be `3:'Zipf distribution'`

File: sad-process-db.py [hunk 3]
```diff
@@ -88,27 +78,10 @@ def process_results(data_dir, dataset_name, results, value_type):
         values = site_results[3:]
         counter = 0
         
-
+        models = {0: 'Logseries', 1:'Poisson lognormal', 2:'Negative binomial', 3:'Negative binomial'}
```
Comment (ethanwhite) [inline sad-process-db.py]: Looks like a duplicate of line 33 (with the same typo).

**Comment by ethanwhite** [comment]
Thanks Xiao. This is a great round of clean up. Much appreciated.

**Comment by rueuntal** [comment]
Thanks for catching all the bugs!

**Comment by ethanwhite** [comment]
Code review FTW!

**Comment by davharris** [comment]
I've read through `sad-comparisons.py`, and I don't see any problems, although it's hard to keep track of the order of various objects (e.g. `results1` versus `results2` versus `results3` and whether the different methods are always in the correct order). I'll try to give it one more read-through with special attention to that.

**Comment by ethanwhite** [comment]
I'm going to try to find time this week to refactor some of that code so that it's more readable to other folks as well - is that OK with you @embaldridge?

**Comment by davharris** [comment]
I found an inconsistency in `sad-process-db.py`:

`names = ['site', 'S', 'N', 'logseries', 'pln', 'negbin', 'AICc_zipf'], delimiter = ",", missing_values = '', filling_values = '')`

Should the zipf column have AICc when the others don't?

**Comment by ethanwhite** [comment]
Good catch @davharris. Can you open that as a separate issue.

**Comment by davharris** [comment]
Sure.  I haven't found any other problems.

**Comment by embaldridge** [comment]
I started to try to make it prettier last year but broke it instead due to coding with broken brain, so I scrapped it and decided to go with the "if it ain't broke, don't fix it" philosophy.  If folks with working brains want to fix it and make it more readable to other folks, I am all on board with that.

**Pull request merged**